overview/overview-business-basics.md

```diff
@@ -0,0 +1,22 @@
+---
+slug: /posdealers/overview/overview-business-basics
+title: overview business basics
+---
+
+# Overview Business Basics
+
+:::info summary
+
+After reading this, you can explain fiskaltrust's business basics.
+
+:::
+
+## Introduction
+
+This **Business Basics** section of our PosDealers documentation overviews our POS solution for checkout and receipt processes. PosDealers can offer their customers (PosOperators) Compliance-as-a-Service and more with our solution. This section lists the supported countries and services. In addition, you will find more details about our management hub for PosDealers, the fiskaltrust.Portal and the fiskaltrust.Middleware. Further, you will find details about our Fair Use Policy.
```

Comment:
maybe instead of: Further, you will find details about our Fair Use Policy.
It also includes details about our Fair Use Policy.

Reply:
unified the whole passage with "You, as a Posdealer, can"